src/controllers/DialogController.ts

```diff
@@ -0,0 +1,23 @@
+import { Request as ExpressRequest, Response } from "express";
+import { Schema } from "mongoose";
+
+import CreateDialogService from "../services/CreateDialogService";
+import ConsultDialogService from "../services/ConsultDialogService";
+
+interface Request extends ExpressRequest {
+  userId?: Schema.Types.ObjectId;
+}
+
+class DialogController {
```

Comment:
is creating this with classes really better? I am particularly an advocate of roles